mkosi/sandbox.py

```diff
@@ -438,6 +443,67 @@ def is_relative_to(one: str, two: str) -> bool:
     return os.path.commonpath((one, two)) == two
 
 
+def pack_file_descriptors() -> int:
+    fds = []
+
+    with os.scandir("/proc/self/fd") as it:
```

Comment:
The whole thing needs the list to be sorted, right? Maybe I'm too paranoid, but does scandir guarantee the order to be lexicographical? If not, maybe a `sorted` afterwards wouldn't hurt?